test/functional/rpc_psbt.py

```diff
@@ -228,6 +228,53 @@ def test_sighash_mismatch(self):
 
         wallet.unloadwallet()
 
+    def test_sighash_adding(self):
+        self.log.info("Test adding of sighash type field")
+        self.nodes[0].createwallet("sighash_adding")
+        wallet = self.nodes[0].get_wallet_rpc("sighash_adding")
+        def_wallet = self.nodes[0].get_wallet_rpc(self.default_wallet_name)
+
+        addr = wallet.getnewaddress(address_type="bech32")
+        outputs = [{addr: 1}]
+        outputs.append({wallet.getnewaddress(address_type="bech32m"): 1})
```

Comment:
nit: `addr` is never used anywhere else, could directly construct the `outputs` object in one go

Reply:
Done